lib/exec.ts

```diff
@@ -91,6 +91,17 @@ export function executeDirect(
         if (command.startsWith('./')) command = path.join(process.cwd(), command);
     }
 
+    const unbufferPrefix = execProps<string>('unbufferStdout', undefined);
+    if (unbufferPrefix) {
+        const splitPrefix = unbufferPrefix.split(' '); // by default ['stdbuf', '-o0']
+        const argsPrefix = splitPrefix.slice(1); // by default ['-o0']
+        if (fs.existsSync(splitPrefix[0])) {
```

Comment:
I don't think this works: `fs.existsSync("stdbuf")` will return `false` unless `stdbuf` is in the current directory, which it won't be.

I think you'll need to use `which` (see e.g. `compile.ts`'s code, something like:

```ts
if (!path.isAbsolute(command)) {
  command = await which(command);
}
```
etc

and/or do it at start up only.